### PR TITLE
Optimize the debug log that affects performance, and unify the style

### PR DESCRIPTION
**Pull request by liudezhi2098**
Master Issue: #13497
### Motivation
Optimize the debug log that affects performance, and unify the style

### Modifications
add isDebugEnabled method

```java
 if (log.isDebugEnabled()) {
    log.debug();
}
```

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: ( no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)
  
### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 
  
- [x]  `no-need-doc`

### CODE REVIEW

File: pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/windowing/WindowManager.java [hunk 1]
```diff
@@ -105,7 +105,9 @@ public void add(T event, long ts, Record<?> record) {
     public void add(Event<T> windowEvent) {
         // watermark events are not added to the queue.
         if (windowEvent.isWatermark()) {
-            log.debug(String.format("Got watermark event with ts %d", windowEvent.getTimestamp()));
+            if (log.isDebugEnabled()) {
+                log.debug(String.format("Got watermark event with ts %d", windowEvent.getTimestamp()));
```
Comment (Jason918) [inline pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/windowing/WindowManager.java:109]: No need to use String.format, just 
log.debug("Got watermark event with ts {}", windowEvent.getTimestamp());

File: pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/windowing/WindowManager.java [hunk 2]
```diff
@@ -145,8 +147,10 @@ public boolean onTrigger() {
         prevWindowEvents.clear();
         if (!events.isEmpty()) {
             prevWindowEvents.addAll(windowEvents);
-            log.debug(String.format("invoking windowLifecycleListener onActivation, [%d] events in "
-                    + "window.", events.size()));
+            if (log.isDebugEnabled()) {
+                log.debug(String.format("invoking windowLifecycleListener onActivation, [%d] events in "
```
Comment (Jason918) [inline pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/windowing/WindowManager.java:151]: Same here

File: pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/windowing/WindowManager.java [hunk 3]
```diff
@@ -216,7 +220,9 @@ private void track(Event<T> windowEvent) {
             lock.unlock();
         }
         eventsSinceLastExpiry.set(0);
-        log.debug(String.format("[%d] events expired from window.", eventsToExpire.size()));
+        if (log.isDebugEnabled()) {
+            log.debug(String.format("[%d] events expired from window.", eventsToExpire.size()));
```
Comment (Jason918) [inline pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/windowing/WindowManager.java:224]: Same here.

File: pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/windowing/WindowManager.java (hunk 1 above)
Comment (liudezhi2098) [reply pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/windowing/WindowManager.java]: yes , I will delete

**Review by Jason918 — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**